packages/action-group/src/ActionGroup.ts

```diff
@@ -363,6 +363,7 @@ export class ActionGroup extends SizedMixin(SpectrumElement, {
 
     private manageChildren(changes?: PropertyValues): void {
         this.buttons.forEach((button) => {
+            button.toggles = false;
```

Comment:
This could cause an issue when a consumer leverages an Action Group without `selects` but Action Buttons with `toggles` in order to manage the selection of elements in the group via some custom algorithm.

I think with the change to Action Button, that we want to move away from listened on `click` and to listening on `change` and then `event.stopPropagation()`ing the child event. Much like we see in [Radio Group](https://github.com/adobe/spectrum-web-components/blob/main/packages/radio/src/RadioGroup.ts#L112-L116).

Reply:
@Westbrook Does that mean we want to dispatch a change event for all clicks on action buttons?

Reply:
Indeed, this is a bit more complex than Radio Group.

The context is that _if_ the Action Group should dispatch `change` events (selects === 'single' | 'multiple') we shouldn't prevent child Action Buttons from dispatching `change` events (`button.toggles = false;`). In that context we should listen for those `change` events and `event.stopPropagation()` on them. We likely also want to `event.preventDefault()` them, as the Action Button is a controller component of the Action Group in that case and does not get to determine its own state.

Does that make next steps here clear enough?